js/IceStorm/weather/sensor.ts

```diff
@@ -0,0 +1,63 @@
+// Copyright (c) ZeroC, Inc.
+
+import { Ice, IceStorm } from "@zeroc/ice";
+import { ClearSky } from "./WeatherStation.js";
+import process from "node:process";
+
+// Create an Ice communicator to initialize the Ice runtime. The communicator is disposed before the program exits.
+await using communicator = Ice.initialize(process.argv);
```

Comment:
Does the communicator get destroyed on Ctrl+C?

Reply:
It does not. The default Ctrl+C handler just call exit.